README.md

```diff
@@ -17,7 +17,7 @@ Open Service Broker.
 
 You'll need:
 
-- A running [kubernetes](https://github.com/kubernetes/kubernetes) cluster
+- A running [kubernetes](https://github.com/kubernetes/kubernetes) or [openshift](https://github.com/openshift/origin/) cluster
```

Comment:
Openshift is kubernetes

Reply:
You have specific instructions for openshift below.  Should I change this to say or openshift's distribution of kubernetes?